fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/RepositoryInitializer.java

```diff
@@ -63,10 +65,12 @@ public void initialize() {
         final PersistentStorageSession session = this.sessionManager.getSession("initializationSession" +
                                                                                  System.currentTimeMillis());
 
-        if (!FEDORA_TO_OCFL_INDEX_FILE.exists()) {
+        final File fedoraToOcflIndexFile = new OCFLConstants().getFedoraToOCFLIndexFile();
```

Comment:
I agree about instantiating the OCFLConstants class... but implementing the injecting is not working out. If you would like to create a pull-request against this pull-request... I will gladly merge it in; otherwise, I suggest we live with the slight funniness.

Reply:
Was there a reason this couldn't remain as static methods?

Reply:
If not, then https://github.com/awoods/fcrepo4/pull/11

Reply:
@whikloj : The static methods prevented setting the OCFL directories in the integration test:
https://github.com/fcrepo4/fcrepo4/pull/1639/files#diff-f210045277d54842209f1d5631554535R62

No matter how I tried to feed the updated OCFL directories into System or Env properties, the Maven values were always picked up.
https://github.com/fcrepo4/fcrepo4/blob/master/pom.xml#L510-L512